drivers/ZW_Dimmer_Rotary/driver.settings.compose.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "type": "group",
+    "label": {
+      "en": "Button behaviour",
+      "nl": "Gedrag knoppen"
+    },
+    "collapsed": false,
+    "children": [
+      {
+        "$extends": "externalSwitchInclusion",
+        "$id": "external_switch_inclusion"
+      }
+    ]
+  },
+  {
+    "type": "group",
+    "label": {
+      "en": "Dimming",
+      "nl": "Dimmen"
+    },
+    "collapsed": true,
+    "children": [
+      {
+        "$extends": "dimDuration",
+        "$id": "dim_duration"
+      },
+      {
```

Comment:
The rotary dimmer does not support configuring minimum brightness (it's configured by knobs on the switch itself).